code/modules/vehicles/unmanned/unmanned_vehicle.dm

```diff
@@ -41,7 +41,7 @@
 	/// If something is already controlling the vehicle
 	var/controlled = FALSE
 	/// Flags for unmanned vehicules
-	var/unmanned_flags = OVERLAY_TURRET|HAS_LIGHTS|UNDERCARRIAGE
+	var/unmanned_flags = OVERLAY_TURRET|HAS_LIGHTS
```

Comment:
This isn't used anywhere else, so if you're going to remove it you might as well remove the define and proc.
That said, removing this does have the knock on effects of a bunch of other shit no longer going over RC's, so I don't think it's the best fix.

I would just add an early return to CanAllowThrough() to disallow mobs.

Reply:
I believe that UAVs aren't _that_ small to go over everything.
My logic for keeping the flag around was in case we wanted to do smaller vehicles. Feels like a waste to remove code that might be used later.

Reply:
fair enough, however my second point still stands, I would just make a check specifically so mobs can't just walk onto it, so runners can bump slash it but you can still throw shit over it/etc.

Reply:
Eh, sure.